llvm/test/tools/llvm-cas-object-format/check-deterministic-flatv1.test

```diff
@@ -0,0 +1,12 @@
+RUN: rm -rf %t && mkdir -p %t
+RUN: llc %S/Inputs/main.ll --filetype=obj -o %t/main.o
+
+// Ingest multiple times and check that we got the same CAS ID for all.
+RUN: llvm-cas-object-format --cas %t/cas %t/main.o -silent -ingest-schema=flatv1 > %t/1.id
```

Comment:
Fine for me as-is for now, but it'd be nicer to have a command-line option in `llvm-cas-object-format` like `-repeat-ingest=4` that does this. Then it's easy to add to any test. Maybe we'd even want to make the default be 2 or 5 or something when in asserts mode.

Reply:
for clarity, by "does this" I mean "calls the ingest function N times for each blob and compares CASIDs to the first result"

Reply:
Good idea!

Reply:
This is done. With assertions enabled by default it will repeat the ingestion one time and check that the ID is the same.